Buildings/Fluid/Movers/UsersGuide.mo

```diff
@@ -437,62 +446,276 @@ If the motor is cooled by the fluid, as indicated by
+as an example.
+</li>
+<li>
+<code>Power_VolumeFlowRate</code> - An array of power vs. <i>V&#775;</i> is provided.
+During simulation, the power is interpolated or extrapolated by
+<a href=\"Modelica://Buildings.Fluid.Movers.BaseClasses.Characteristics.power\">
+Buildings.Fluid.Movers.BaseClasses.Characteristics.power</a>.
+The efficiency is then computed from the power.
+See
+<a href=\"Modelica://Buildings.Fluid.Movers.Validation.PowerExact\">
+Buildings.Fluid.Movers.Validation.PowerExact</a>
+as an example.
+</li>
+<li>
+<code>EulerNumber</code> - An efficiency, <i>&Delta;p</i>, and
+<i>V&#775;</i> are provided that correspond to an operating point where the efficiency is at its peak.
+During simulation, the efficiency and power are computed using the package
+<a href=\"Modelica://Buildings.Fluid.Movers.BaseClasses.Euler\">
+Buildings.Fluid.Movers.BaseClasses.Euler</a>.
+The model finds the efficiency by evaluating the following correlation:
+<br/>
+<p align=\"center\">
+<img alt=\"image\" src=\"modelica://Buildings/Resources/Images/Fluid/Movers/BaseClasses/Euler/eulerCorrelation.svg\"/>
+<br/>
+</p>
+where <i>y=&eta; &frasl; &eta;<sub>p</sub></i>,
+<i>x=log10(Eu &frasl; Eu<sub>p</sub>)</i>,
+with the subscript <i>p</i> denoting the condition where
+the mover is operating at peak efficiency.
+The Euler number is defined as
+<br/>
 <p align=\"center\" style=\"font-style:italic;\">
-  &eta;<sub>hyd</sub> = 1,<br/>
-  &eta;<sub>mot</sub> = &eta;.
+Eu=(pressure forces)/(inertial forces)
 </p>
+from which one can derive the ratio of Euler numbers as<br/>
+<p align=\"center\" style=\"font-style:italic;\">
+Eu &frasl; Eu<sub>p</sub>=(&Delta;p&nbsp;V&#775;<sub>p</sub><sup>2</sup>)
+&frasl; (&Delta;p<sub>p</sub>&nbsp;V&#775;<sup>2</sup>).
+</p>
+<p>
+For more information, see
+<a href=\"Modelica://Buildings.Fluid.Movers.BaseClasses.Euler.correlation\">
+Buildings.Fluid.Movers.BaseClasses.Euler.correlation</a>
+and <a href=\"https://energyplus.net/assets/nrel_custom/pdfs/pdfs_v9.6.0/EngineeringReference.pdf\">
+EnergyPlus 9.6.0 Engineering Reference</a>
+chapter 16.4 equations 16.209 through 16.218.<br/>
+See
+<a href=\"Modelica://Buildings.Fluid.Movers.Validation.PowerEuler\">
+Buildings.Fluid.Movers.Validation.PowerEuler</a>
+as an example.
+</li>
+<li>
+<code>NotProvided</code> - The information of this efficiency term is not provided.
+During simulation, it will be computed using the other efficiency terms.
+</li>
+</ul>
 <p>
-However, if <code>per.use_powerCharacteristic=false</code>, then
-performance data for
-<i>&eta;<sub>hyd</sub></i> and
- <i>&eta;<sub>mot</sub></i> need to be provided by the user, and hence
-the model computes
+<i>&eta;<sub>mot</sub></i> can be computed using the following choices:
 </p>
-<p align=\"center\" style=\"font-style:italic;\">
-  &eta; = &eta;<sub>hyd</sub> &nbsp; &eta;<sub>mot</sub><br/>
-  P<sub>ele</sub> = W<sub>flo</sub> &frasl; &eta;.
+<ul>
+<li>
+<code>Efficiency_VolumeFlowRate</code>
```

Comment:
This item misses the description, please add it.

Reply:
Added in df96e8233e3b1cc7bb2fd7285a1813900805191f.